internal/exec/terraform_outputs.go

```diff
@@ -296,19 +297,15 @@ func GetTerraformOutput(
 
 	sections, err := ExecuteDescribeComponent(component, stack, true, true, nil)
 	if err != nil {
-		if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-			fmt.Printf("\r✗ %s\n", message)
-		}
-		l.Fatal("Failed to describe the component", "component", component, "stack", stack, "error", err)
+		u.PrintMessage(fmt.Sprintf("\r✗ %s\n", message))
```

Comment:
Does this go to stderr?

Reply:
> Does this go to stderr?

https://github.com/cloudposse/atmos/blob/main/tests/test-cases/atmos-functions.yaml#L21-L26
Based on above test case we do not add the line to stderr.

Reply:
There are 3 problems with this:
1) engineer that implemented it emitted it to stdout, which broke all of our GitHub actions
2) engineer that "fixed" it changed it to only emit on debug, which still break our GitHub actions, just it's not the default log level
3) the whole purpose of this is not for logging or debugging, so (2) broke the usefulness of the feature

The appropriate fix is to emit output that is part of TUI to stderr so stdout is untarnished.